### PR TITLE
Riten_Assignment_11

### PR DESCRIPTION
**Pull request by ritenthakkar**

### CODE REVIEW

File: src/riten/PrimeWhileCheck.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+				// System.out.println(number+" Num is not Prime");
+				status = false;
+				break;
+			}
+			index2++;
+		}
+		if (status) {
+			// System.out.println(number + " Num is Prime Number");
+
+			boolean flag = true;
```
Comment (surabhi-9) [inline src/riten/PrimeWhileCheck.java:24]: no flag variable and if condition needed, directly we can check if(status), check if(sum<=range) then call sumPrime method.

**Review by surabhi-9 — APPROVED** [review]
rest code looks good to me

File: src/riten/PrimeWhileCheck.java (hunk 1 above)
Comment (ritenthakkar) [reply src/riten/PrimeWhileCheck.java:24]: Yes Yes. I will make changes in code and commit the same. you can then please merger the code to master.

Comment (surabhi-9) [reply src/riten/PrimeWhileCheck.java:24]: its merged already. just added comment so that you can consider in further programming. :)

**Pull request merged**